src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -151,22 +151,22 @@ const WorkflowDashboard = _.flow(
             ])
           ]),
           makeSection('Links', [
-            div({ style: { display: 'flex', flexFlow: 'row wrap', marginTop: '0.5rem' } }, [
+            div({ style: { display: 'flex', flexFlow: 'row wrap', marginTop: '0.5rem', lineHeight: '2rem' } }, [
               h(Link, {
                 ...Utils.newTabLinkProps,
                 href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
-                style: { display: 'flex' },
+                style: { display: 'flex', marginRight: '2em', alignItems: 'center' },
```

Comment:
Nit: Because there are more styles being applied to these than just flex now, I would say we should probably pull out to a variable.

Reply:
OBE: I reverted to closer to how they were